samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatController.cs

```diff
@@ -62,18 +66,31 @@ public ChatController(ILogger<ChatController> logger)
     public async Task<IActionResult> ChatAsync(
         [FromServices] IKernel kernel,
         [FromServices] CopilotChatPlanner planner,
-        [FromBody] Ask ask,
+        [FromServices] AskConverter askConverter,
```

Comment:
I think we don't necessarily need to use the `Ask` for this API since it only invokes the `ChatSkill`. We can create a new model for the body that contains the ChatId and the message. Would love @adrianwyatt's take on this.

Reply:
I agree, but such a change is outside the intended scope of this PR. The goal of the PR is only to close the most critical security gap.